config/crd/patches/webhook_in_ibmpowervsclustertemplates.yaml

```diff
@@ -1,17 +1,19 @@
 # The following patch enables conversion webhook for CRD
 # CRD conversion requires k8s 1.13 or later.
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: ibmpowervsclustertemplates.infrastructure.cluster.x-k8s.io
 spec:
   conversion:
     strategy: Webhook
-    webhookClientConfig:
-      # this is "\n" used as a placeholder, otherwise it will be rejected by the apiserver for being blank,
-      # but we're going to set it later using the cert-manager (or potentially a patch if not using cert-manager)
-      caBundle: Cg==
-      service:
-        namespace: system
-        name: webhook-service
-        path: /convert
+    webhook:
+      conversionReviewVersions: ["v1", "v1beta1"]
```

Comment:
why is this `["v1", "v1beta1"]`? why not `["v1beta1", "v1beta2"]`

Reply:
will spend some more time on this, fix as part of a separate PR